src/main/java/graphql/execution/DataFetcherExceptionHandler.java

```diff
@@ -41,8 +23,5 @@ default DataFetcherExceptionHandlerResult onException(DataFetcherExceptionHandle
      *
      * @return a result that can contain custom formatted {@link graphql.GraphQLError}s
      */
-    default CompletableFuture<DataFetcherExceptionHandlerResult> handleException(DataFetcherExceptionHandlerParameters handlerParameters) {
-        DataFetcherExceptionHandlerResult result = onException(handlerParameters);
-        return CompletableFuture.completedFuture(result);
-    }
+    CompletableFuture<DataFetcherExceptionHandlerResult> handleException(DataFetcherExceptionHandlerParameters handlerParameters);
```

Comment:
You MUST now implement the method if you make your own.  Its no longer defaulted